gpkit/small_scripts.py

```diff
@@ -3,6 +3,27 @@
 from .small_classes import Strings, Quantity
 
 
+def try_str_without(el, excluded):
+    if hasattr(el, "str_without"):
+        return el.str_without(excluded)
+    else:
+        return str(el)
+
+
+def veckeyed(key):
+    vecdescr = dict(key.descr)
+    del vecdescr["idx"]
+    vecdescr.pop("value", None)
+    return key.__class__(**vecdescr)
+
+
+def listify(l):
+    if isinstance(l, Iterable):
+        return list(l)
+    else:
+        return [l]
+
+
 def isequal(a, b):
     if (isinstance(a, Iterable) and
             not isinstance(a, Strings+(tuple, list, dict))):
```

Comment:
For all of these in small_scripts, I don't know. Possibly? I put them here because they seemed general enough that something else might use them.